index.js

```diff
@@ -202,6 +202,23 @@ async function iterateOverCollectionDefinitionAndRegisterCruds(fastify) {
   }
 }
 
+async function registerHelperRoutes(fastify) {
+  registerGetSchemasRoute(fastify)
+}
+
+async function registerGetSchemasRoute(fastify) {
+  const jsonSchemasMap = {}
+  fastify.get('/schemas', () => {
```

Comment:
Great catch! We've solved this by adding the env var HELPERS_PREFIX defaulted to `/-/` like the status probes. This means that by default the URL is `/-/schemas` but the prefix can be changed by the used to avoid colisions.